reference/orders.v2.oas2.yml

```diff
@@ -3082,20 +3082,19 @@ components:
               tax_provider_id:
                 type: string
                 description: |
-                  BasicTaxProvider - Tax is set to manual.
+                  BasicTaxProvider - Tax is set to manual and order is created in the store.
```

Comment:
@bc-tgomez are we sure the order has to be created in the store here? I'm not privy to your research. Could be good to check with engineering if you haven't already.

Reply:
From what I had discussed with [Willem](https://bigcommerce.slack.com/archives/C96P5T3J4/p1643313600099500) that was the rule. It populates at order creation by any means except `/v2/orders`.

Reply:
👍 that's correct